server/models/users.js

```diff
@@ -0,0 +1,7 @@
+const db = require('../db');
```

Comment:
Hey Russ,

did you want to go ahead and adjust it for the db folder to be located in root?

`const db = require('../../db')`